WordPress/Classes/ViewRelated/Pages/PageListCell.swift

```diff
@@ -35,14 +42,11 @@ final class PageListCell: UITableViewCell, Reusable {
     override func prepareForReuse() {
         super.prepareForReuse()
 
-        cancellables = []
         imageLoader.prepareForReuse()
```

Comment:
Let's remove the call to `imageLoader.prepareForReuse()` from `configure(with:)`, since we're calling it here already.